NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,29 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, IEnumerable<ECPubKey> pubkeys, out ECPubKey? combinedPubKey)
```

Comment:
This implementation is not following line by line 

```c
int secp256k1_ec_pubkey_combine(const secp256k1_context* ctx, secp256k1_pubkey *pubnonce, const secp256k1_pubkey * const *pubnonces, size_t n) {
    size_t i;
    secp256k1_gej Qj;
    secp256k1_ge Q;

    ARG_CHECK(pubnonce != NULL);
    memset(pubnonce, 0, sizeof(*pubnonce));
    ARG_CHECK(n >= 1);
    ARG_CHECK(pubnonces != NULL);

    secp256k1_gej_set_infinity(&Qj);

    for (i = 0; i < n; i++) {
        secp256k1_pubkey_load(ctx, &Q, pubnonces[i]);
        secp256k1_gej_add_ge(&Qj, &Qj, &Q);
    }
    if (secp256k1_gej_is_infinity(&Qj)) {
        return 0;
    }
    secp256k1_ge_set_gej(&Q, &Qj);
    secp256k1_pubkey_save(pubnonce, &Q);
    return 1;
}
```

Please use array instead of IEnumerable. (so we can get `n` from it, without having to evaluate the enumerable)

Reply:
Ok 
> Use variable name as well.

You mean same variable name as much as possible right?